setup.py

```diff
@@ -62,6 +62,6 @@ def read(fname):
                  'mock', 'sagemaker==1.28.1', 'docker-compose', 'boto3==1.10.32', 'six==1.13.0',
                  'python-dateutil>=2.1,<2.8.1', 'botocore==1.13.32', 'awscli==1.16.278',
                  'requests_mock==1.7.0'],
-        ':python_version=="3.6"': ['sagemaker-experiments==0.1.7']
+        ':python_version>="3.5"': ['sagemaker-experiments==0.1.7']
```

Comment:
Looks like you're not installing this as part of a test dependency. We use 

```
pip install -e.[test]
```

to install the test dependencies.

Reply:
looking at sagemaker_helper.py in the build package tox.ini is not used to manage dependencies. so a solution will either have to modify the build logic or add support for 2.x to the dependency

Reply:
Hi Dana! Do you think something like this could work?

```
Import sys

test_dependencies = ['tox', ....]

if sys.version_info.major > 2:
    test_dependencies.append('sagemaker-experiments==0.1.7')

setup(
    .........
    extras_require={
        'test': test_dependencies
    },
)
```

Reply:
ya that looks like it would work

Reply:
updated PRs